app/com/yetu/oauth2provider/views/setup/startSignUpForSetup.scala.html

```diff
@@ -116,13 +122,7 @@
 		</div>
 		@views.html.setup.footer()
         <script src="@controllers.routes.Assets.versioned("javascripts/formMessageController.js")"></script>
-        <script>
-            var hoverHelp = function(element){
-                element.setAttribute('class', 'help-block');
-            };
-            var blurHelp = function(element){
-                element.setAttribute('class', 'help-block display-none');
-            };
-        </script>
+        <script src="@controllers.routes.Assets.versioned("javascripts/trimEmail.js")"></script>
+        <script src="@controllers.routes.Assets.versioned("javascripts/oneErrorMessageForPassword.js")"></script>
```

Comment:
we should set up a proper process to load and minify the javascript all from one file. It is starting to become messy.